### PR TITLE
CB-7182 IT fails intermittently

### PR DESCRIPTION
**Pull request by bbihari**
cherry-pick

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/action/v4/environment/AbstractEnvironmentAction.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                LOGGER.info("Exception during executing Environment action: ", e);
+                if (e.getMessage().contains("flow under operation")) {
+                    waitTillFlowInOperation(testDto.getWaitUtil());
+                    retries++;
+                } else {
+                    throw e;
+                }
+            }
+        }
+        throw new Exception("Exception during executing Environment action: exceeding maxrety during waiting for flow");
```
Comment (topolyai5) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/action/v4/environment/AbstractEnvironmentAction.java:32]: typo in `maxrety`

Comment (bbihari) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/action/v4/environment/AbstractEnvironmentAction.java:32]: good catch! :)

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/FreeIpaStartStopTest.java [hunk 2]
```diff
@@ -20,6 +22,8 @@
 
 public class FreeIpaStartStopTest extends AbstractIntegrationTest {
 
+    private static final Duration POLLING_INTERVAL = Duration.of(3000, ChronoUnit.MILLIS);
```
Comment (lnardai) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/FreeIpaStartStopTest.java:25]: Unused variable?

Comment (bbihari) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/FreeIpaStartStopTest.java]: fixed

**Pull request merged**